diff -urN tedia2sql-orig/tedia2sql tedia2sql/tedia2sql
--- tedia2sql-orig/tedia2sql	2004-01-08 01:22:30.000000000 +0100
+++ tedia2sql/tedia2sql	2004-01-08 01:34:14.000000000 +0100
@@ -108,6 +108,7 @@
 my $sqlCommentString = "--";
 my $goCommand = ";";
 my $sqlIndent = "  ";
+my $version = "1.2.9";
 my $versionInfo = "tedia2sql -- v1.2.9";
 my $authorInfo = "See http://tedia2sql.tigris.org/AUTHORS.html for tedia2sql author information";
 my $verbose = undef;
@@ -1758,13 +1759,12 @@
 	#
 	# if rcfile has PARAM .= VAL then @PARAM will exist with
 	# each successive element of VAL being PUSHed onto the array
-	if (open (RCFILE, "< $rcFileName")) {
+	my $homeFile = $ENV{HOME} . "/.tedia2sqlrc";
+	if (open (RCFILE, "< $homeFile")) {
 		# all is cool
-	} elsif ($rcFileName =~ /.*?\/(\w+)$/) {
-		my $tryThisFile = $ENV{HOME} . "/.$1";
-		open (RCFILE, "< $tryThisFile") || die "Can't open $rcFileName or $tryThisFile";
-	} else {
-		die "Can't open $rcFileName and can't determine what the rcfile should be...";
+	}
+	else {
+		die "Can't open $homeFile, gunzip it from /usr/share/doc/tedia2sql-$version/tedia2sqlrc.gz";
 	}
 
 	push (@{$cfg->{_rcFileName}}, {file=>$rcFileName,date=>scalar (localtime())});
